cmd/leo/create/create.go

```diff
@@ -47,6 +48,7 @@ func New(cloud *common.Cloud) *cobra.Command {
 	cmd.Flags().StringVar(&o.Machine, "machine", "m", "machine type")
 	cmd.Flags().StringVar(&o.Name, "name", "", "deterministic name")
 	cmd.Flags().StringVar(&o.Output, "output", "", "output directory to download")
+	cmd.Flags().StringSliceVar(&o.Exclude, "exclude", nil, "comma-separated list of patterns to exclude from uploading and downloading")
```

Comment:
Worth updating HCL config parsing “logic” here?

https://github.com/iterative/terraform-provider-iterative/blob/843017a40e3eafc4182e54a2c5b97a565d58b29b/cmd/leo/root.go#L90-L140

Reply:
I try to avoid that block of code (gives me nightmares), but sure..

Reply:
true